### PR TITLE
Virology Expansion Pack: You'll Only Make These After Shuttle Call Edition

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
add: You can now make level 7 symptoms with Conductive Mutagenic Agar, made with gold and mutagenic agar. Previous levels 7-8 are now 8-9.
add: There are now several new high-level symptoms. Don't catch em all.
/:cl:

Adds an intermediate virology level: level 7 is now made with gold and mutagenic agar, while previous levels 7 and 8 are now 8 and 9. So there is something inbetween "you have it roundstart" and "you need 15 bars of uranium".

Adds Bluespace Instability, which causes the mob to randomly teleport as if it crushed a (smaller) bluespace crystal. (Lv7)
Adds Cluster Headache, which causes the mob to feel intense headaches. (i.e. short stuns + some brain damage). (Lv5)
Adds Radioactive Metabolism, which causes the mob to rarely emit radioactive pulses. (Lv8)
Adds Mithocondrial Depolarization, which causes the mob to be extremely cold. (Lv5)
Adds Overloaded Nerves, which will shocks the mob with Teslium. (Lv7)
Adds Viral Electric Net, which will cause the mob to shoot short-ranged tesla bolts and EMP pulses. The mob is not immune to his own shocks unless it wears gloves/hardsuit. (Lv9)
Adds Hormonal Hypertrophy, which causes the mob to receive random buffs/debuffs connected to a particular emotion. (Lv8)

I'll need feedback on the balance of these symptoms, of course.

### CODE REVIEW

**Comment by Cheridan** [comment]
Please stop using the term lategame as if we're an MMO and not a round-based game that resets every 30 minutes :mask:

File: code/datums/diseases/advance/symptoms/stimulant.dm [hunk 1]
```diff
@@ -35,4 +35,87 @@ Bonus
...
+							M.reagents.add_reagent("bath_salts", 5)
+						M.SetStunned(0)
+						M.SetParalysis(0)
+						M.SetWeakened(0)
+						M.SetSleeping(0)
+						M.hallucination += 30
+						M.emote("scream")
+
+					if(5,6) //Determination: Lets you tough out your wounds. You ain't dying today.
+						M << "<span class='notice'>You are filled with determination.</span>"
```
Comment (GunHog) [inline code/datums/diseases/advance/symptoms/stimulant.dm]: -_-

**Comment by TheColdTurtle** [comment]
seems like this is just more shit to get a virologist lynched

**Comment by Niknakflak** [comment]
seems OP and dumb

**Comment by kevinz000** [comment]
Now I can walk around and kill everyone by being a living tesla coil ;D

**Comment by Xhuis** [comment]
Some of these seem pretty strong or like they wouldn't be used. The emotion one in particular is insane, and I'm not a fan of ones that let you hurt everyone around you.

**Comment by Banana-Fist** [comment]
Goodness no. Don't buff virology right now.

**Pull request closed without merging**